doc/RecordingControl.xml

```diff
@@ -430,7 +430,12 @@ Change Request 2061, 2063, 2065, 2109</revremark>
         <para>
           <emphasis role="bold">Mode</emphasis>: If it is idle, nothing shall happen. If it is active and the recording job has the highest priority, the device shall try to obtain data from the receivers. A client shall use GetRecordingJobState to determine if data transfer is really taking place. The only valid values for Mode shall be “Idle” and “Active”.</para>
         <para>
-          <emphasis role="bold">Priority</emphasis>: This shall be a non-negative number. If there are multiple recording jobs that store data to the same track, the device will only store the data for the recording job with the highest priority. The priority is specified per recording job, but the device shall determine the priority of each track individually. If there are two recording jobs with the same priority, the device shall record the data corresponding to the recording job that was activated the latest.</para>
+          <emphasis role="bold">Priority</emphasis>: This shall be a non-negative number. If there
+          are multiple recording jobs that store data to the same track, the device shall only store
+          data for the recording job with the highest priority. The priority is specified per
+          recording job, but the device shall determine the priority of each track individually. If
+          there are multiple recording jobs with the same highest priority it is undefind which of
+          them is activated.</para>
         <para>The value 0 indicates the lowest priority. Higher values shall indicate a higher priority.</para>
```

Comment:
@HansBusch reading over the specification there is no definition of a range or Maximum set of values for "Priority". It seams quite arbitrary.